### ISSUE TITLE
Lint: Mypy broken with Python 3.10

### ISSUE THREAD

**Issue by Rogdham** [demisto/demisto-sdk#1831]
Hello,

With Python 3.10, mypy linting is broken. I have the following output when running `demisto-sdk lint -i Packs/redacted`:

```
usage: mypy [-h] [-v] [-V] [more options; see below]
            [-m MODULE] [-p PACKAGE] [-c PROGRAM_TEXT] [files ...]
mypy: error: argument --python-version: Python 3.1 is not supported (must be 3.4 or higher)
```

I believe this is due to the version of Python being passed as a float, meaning that `3.10` will be converted to `3.1`. For example in [commands_builder.py](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/lint/commands_builder.py#L154):
```py
def build_mypy_command(files: List[Path], version: float) -> str:
    ...
```

Version used: `demisto-sdk==1.5.7`

**Comment by dorschw** [comment]
Hi, thank you for opening an issue.

It's a known issue that we're working to solve it as soon as possible. We
intend to release a new version fixing it by next week.

Until then, the temporary solution is to skip linting on 3.10 images,
and/or keep using 3.9 as long as possible.

We're sorry for the inconvenience caused and will update regarding the fix.

**Comment by ShahafBenYakir** [comment]
Hi, the issue was fixed, please update your SDK package or pull from master if working in demisto/content

**Comment by Rogdham** [comment]
Confirmed fixed with `demisto-sdk==1.5.8`, thank you! :tada:
